### ISSUE TITLE
responseType: class, Error executing command: No model found

### ISSUE THREAD

**Issue by kinger-de** [csarrazi/CsaGuzzleBundle#149]
Hi , I would like to direct the return of an operation in a class . But I get the error that the model was not found with the class name . It is always trying to load a model , rather than the class .

My definition of the operation is as follow:
```
"getResources" : {
   "summary:": "Get swagger Resource Listing",
   "uri" : "/resources.json",
   "httpMethod": "GET",
   "responseType": "class",
   "responseClass" : "Loco\\Utils\\Swizzle\\Response\\ResourceListing"
}
```

Does anyone have an idea , or can tell me what I have forgotten.

Thank you.

**Comment by csarrazi** [comment]
Actually, you should not use responseClass, but responseModel instead.

Also, responseModel is actually a sub-node of the `models` array, in the root section of your definition.

Please check the up-to-date documentation of guzzle-services. I'm closing this as it is not related to the bundle at all, but to the use of `guzzlehttp/guzzle-services`.

Check https://github.com/guzzle/guzzle-services/blob/8adb1d160b19eeb8320d2f846bc3b8dac8a878a4/README.rst for more information.

**Comment by csarrazi** [comment]
By the way, only two types of models are supported:

* array, which deserialises the response as a PHP array
* object, which deserialises the response as a PHP object (StdClass).

It seems that you are reading the documentation for the old version of guzzle-services, only compatible with Guzzle 3.0.

I would rather recommend you used Guzzle 6 instead, and create your own wrapper on top of the Guzzle client, using Symfony's serializer.

**Comment by kinger-de** [comment]
Thank you. And yes , you're right . I have not noticed that the package I wanted to use , based on an old version Guzzle .

**Comment by csarrazi** [comment]
No problem! 😃

By the way, support for guzzle-services will be re-implemented in release 2.1 or 2.2 (depending on the time frame), once it has been migrated to Guzzle 6.
